plugins/templates/ios-single-view-no-ib/src/main/resources/archetype-resources/robovm.xml

```diff
@@ -12,4 +12,6 @@
             <directory>resources</directory>
         </resource>
     </resources>
+    <!-- Set to 'true' to trigger compilation and linking depending on changes in IDEA -->
```

Comment:
another moment/bug-like: 
please comment  out `<smartSkipRebuild>false</smartSkipRebuild>` as when previous compiler is used (without IC)  it crashes with exception 
`ElementException: Element 'smartSkipRebuild' does not have a match in class org.robovm.compiler.config.Config`

I understand that new compiler will handle it but it is better to keep compatibility with older ones. 
Probably better case would be to have it commented out with `true` state and comment saying to uncomment line bellow to enable

Reply:
Fixed this.